packages/garbo/src/familiar/experienceFamiliars.ts

```diff
@@ -52,7 +63,7 @@ export default function getExperienceFamiliars(): GeneralFamiliar[] {
   return experienceFamiliars
     .filter(
       ({ used, familiar }) =>
-        have(familiar) && !get(used) && familiar.experience < 400,
+        have(familiar) && !used && familiar.experience < 400,
```

Comment:
This issue persists in the latest draft

Reply:
It should evaluate and return as boolean, such that `!used` will evaluate correctly. Am I missing something?

Reply:
Yes: 
1. `!used` will evaluate as a boolean, but if `used` equal to, for example, the string `"_meatifyMatterUsed"`, then the value of that boolean isn't very meaningful.
2. [Because `used` is defined as a boolean (when it's not a property name, that is) instead of a `() => boolean`, it's evaluated exactly once, which will not lead to the behaviors that you want](https://github.com/loathers/garbage-collector/pull/1853/files#r1585715461)

Reply:
Believe I fixed this now?